payments-core/src/main/java/com/stripe/android/payments/core/injection/PaymentLauncherViewModelFactoryComponent.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.payments.core.injection
+
+import android.content.Context
+import com.stripe.android.payments.paymentlauncher.PaymentLauncherViewModel
+import dagger.BindsInstance
+import dagger.Component
+import javax.inject.Named
+import javax.inject.Singleton
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * Component to inject [PaymentLauncherViewModel.Factory] when the app process is killed and
+ * there is no [Injector] available.
+ */
+@Singleton
+@Component(
+    modules = [
+        PaymentLauncherModule::class,
+        StripeRepositoryModule::class
+    ]
+)
+internal interface PaymentLauncherViewModelFactoryComponent {
+    fun inject(factory: PaymentLauncherViewModel.Factory)
+
+    @Component.Builder
+    interface Builder {
+        @BindsInstance
+        fun context(context: Context): Builder
+
+        @BindsInstance
+        fun enableLogging(@Named(ENABLE_LOGGING) enableLogging: Boolean): Builder
+
+        @BindsInstance
+        fun ioContext(@IOContext workContext: CoroutineContext): Builder
+
+        @BindsInstance
+        fun uiContext(@UIContext uiContext: CoroutineContext): Builder
```

Comment:
These are still going to be moved to a separate module so they're not provided from the ViewModel.Factory, right?

Reply:
Yes, will do that in a separate PR